### DISCUSSION TITLE
Robot cu exapnsion hub

### DISCUSSION THREAD

**Discussion by matei8** [Sova-Tech/FTC#29] [General]
Luca a facut asta in fusion

**Comment by LucaVictor** [comment]
Am creat 2 variante in care putem folosi niste pereti exteriori dintr-un policarbonat ca sa montam control si expansion hub ul.

**Varianta 1**

![image](https://user-images.githubusercontent.com/76179120/106381578-70128800-63c2-11eb-8002-efbabffaab56.png)
![image](https://user-images.githubusercontent.com/76179120/106381580-74d73c00-63c2-11eb-897d-f5daa2020770.png)
![image](https://user-images.githubusercontent.com/76179120/106381641-deefe100-63c2-11eb-8331-4d20b50fe502.png)

Policarbonatul este atasat de spatele robotului, adica de acele grinzi de metal(15mm extrusion) de la spatele robotului(poza 3)

**Reply by matei8** [reply]
Cred ca e mai bine sa punem in lateral pereti

**Comment by LucaVictor** [comment]
**Varianta 2**

![image](https://user-images.githubusercontent.com/76179120/106381660-fcbd4600-63c2-11eb-9f45-dc0ff2eadbd3.png)
![image](https://user-images.githubusercontent.com/76179120/106381662-05158100-63c3-11eb-8c71-a6afcd4b2cdf.png)
![image](https://user-images.githubusercontent.com/76179120/106381670-152d6080-63c3-11eb-9fe6-be8889b5fb15.png)

Policarbontaul este atasat de C channel(poza 3)
